Formula/das_tool.rb

```diff
@@ -36,12 +37,15 @@ def install
 
     chmod 0755, "DAS_Tool"
     inreplace "DAS_Tool", "split --numeric-suffixes --lines=", "split -l " if OS.mac?
+    inreplace "src/Fasta_to_Scaffolds2Bin.sh", "#!/bin/env bash", "#!/bin/bash"
```

Comment:
This PR is fine. I'd suggest opening an upstream PR to change `#!/bin/env bash` to either `#!/usr/bin/env bash` which is more portable or `#!/bin/bash` which is simpler and good enough.

Reply:
https://github.com/cmks/DAS_Tool/pull/40